props/prop_base.h

```diff
@@ -1764,6 +1764,10 @@ class PropBase : CommandParser, protected Looper, protected SaberBase, public Mo
       ToggleColorChangeMode();
       return true;
     }
+    if (!strcmp(cmd, "get_variation")) {
+      STDOUT.println(current_preset_.variation);
```

Comment:
This should probably print the variation from SaberBase. (They are probably the same most of the time though.)

Reply:
I had considered that. This was a request from ShtokyD, as he wanted to be able to get current variation via serial on the phone. I kind of assumed to use current preset's. He said it works great like this.

Reply:
The "variation" command uses SaberBase::SetVariation, and unless there is a very good reason for it, this command should be doing the inverse of that.

"it works" is not a good enough reason.

Reply:
Ok. How about now?